src/solvers/dgmulti/sbp.jl

```diff
@@ -244,6 +244,56 @@ function StartUpDG.RefElemData(element_type::Hex,
     M, Pq, Drst, LIFT)
 end
 
+function StartUpDG.RefElemData(element_type::Hex,
+                               D::AbstractPeriodicDerivativeOperator;
+                               tol = 100*eps())
+
+  approximation_type = D
+  N = SummationByPartsOperators.accuracy_order(D) # kind of polynomial degree
+
+  # 1D operators
+  nodes_1d, weights_1d, D_1d, I_1d = construct_1d_operators(D, tol)
+
+  # volume
+  s, r, t = vec.(StartUpDG.NodesAndModes.meshgrid(nodes_1d, nodes_1d, nodes_1d))
```

Comment:
```suggestion
  # volume
  # to match ordering of nrstJ
  s, r, t = vec.(StartUpDG.NodesAndModes.meshgrid(nodes_1d, nodes_1d, nodes_1d))
```
I would wonder why we don't use `r, s, t` otherwise